Converters/Rhino/Speckle.Converters.RhinoShared/ToSpeckle/Raw/HatchToSpeckleConverter.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Speckle.Converters.Common;
+using Speckle.Converters.Common.Objects;
+using Speckle.Objects;
+
+namespace Speckle.Converters.Rhino.ToSpeckle.Raw;
+
+public class HatchToSpeckleConverter : ITypedConverter<RG.Hatch, SOG.Region>
+{
+  private readonly ITypedConverter<RG.Curve, ICurve> _curveConverter;
+  private readonly IConverterSettingsStore<RhinoConversionSettings> _settingsStore;
+
+  public HatchToSpeckleConverter(
+    ITypedConverter<RG.Curve, ICurve> curveConverter,
+    IConverterSettingsStore<RhinoConversionSettings> settingsStore
+  )
+  {
+    _curveConverter = curveConverter;
+    _settingsStore = settingsStore;
+  }
+
+  /// <summary>
+  /// Converts a Hatch geometry to a Speckle Region object.
+  /// </summary>
+  /// <param name="target">The Hatch to convert.</param>
+  /// <returns>The converted Speckle Region object.</returns>
+  public SOG.Region Convert(RG.Hatch target)
+  {
+    // get boundary and inner curves
+    RG.Curve rhinoBoundary = target.Get3dCurves(true)[0];
+    RG.Curve[] rhinoLoops = target.Get3dCurves(false);
+
+    ICurve boundary = _curveConverter.Convert(rhinoBoundary);
+    List<ICurve> innerLoops = rhinoLoops.Select(x => _curveConverter.Convert(x)).ToList();
+
+    List<ICurve> allCurves = new List<ICurve> { boundary }
```

Comment:
this is a bit inefficient, since you are creating a new list with `concat` and then calling `tolist()` again. You can just use an `addrange` to modify your allcurves list.

Actually, you aren't using the allcurves list at all. Just delete lines 35-37